### DISCUSSION TITLE
How to fix the "Unable to Download"/ "Try again later" error on the Mac App Store - uninstall first!

### DISCUSSION THREAD

**Discussion by npaine01** [chidiwilliams/buzz#443] [General]
I saw a review on the internet version of the app store where someone had the error that said, "unable to download" or "try again later". I also had this error when I tried to install it after purchasing it.

I figured out that you needed to uninstall it first (the dmg version that you can install for free) to be able to install the App Store version.

If you have this same error, give it a shot!

**Comment by chidiwilliams** [comment]
Thanks a lot for this. I was pretty confused about that comment.

**Comment by npaine01** [comment]
My pleasure. Your app is pretty fantastic. I’ve recommended it to quite a
few colleagues. As a teacher who is teaching language acquisition, it is
invaluable!

Please let people know if there are things we can do to support
development.

Best Regards,
Nathan Paine
